libs/model/src/aggregates/user/signIn/utils.ts

```diff
@@ -95,6 +97,48 @@ export async function findUserBySso(
   return users[0];
 }
 
+/*
+ Check minimum balance requirement for new or existing users
+ signing in with native wallets that are below SocialVerified tier
+*/
+async function checkNativeWalletBalance(
+  address: string,
+  foundUser: UserAttributes | null,
+  ethChainId?: number,
+  transaction?: Transaction,
+): Promise<UserTierMap> {
+  if (
+    (foundUser?.tier || UserTierMap.IncompleteUser) < UserTierMap.SocialVerified
+  ) {
+    const balances = ethChainId
+      ? await tokenBalanceCache.getBalances({
+          addresses: [address],
+          balanceSourceType: BalanceSourceType.ETHNative,
+          sourceOptions: { evmChainId: ethChainId },
+        })
+      : { [address]: '0' };
+    const balance = BigInt(balances[address] || '0');
+    const minBalance = BigInt(config.CONTESTS.MIN_USER_ETH * 1e18);
```

Comment:
What is the MIN_USER_ETH value currently? @dillchen we should have a non-dust value for this one no?

Reply:
I'm reusing CONTESTS.MIN_USER_ETH since it's a similar concept, but will add another variable if necessary

Reply:
I believe the contest threshold is lower than what we want for sign-in tiering. From a brief thread in all-eng last week it seems we may be for something like 10$ worth of ETH which would be something like 0.006 at current pricing.

It seems that the current MIN_USER_ETH is literally hardcoded to 0 with no env var loading -> `MIN_USER_ETH: 0,`

Reply:
ok, added new var TIER_SOCIAL_VERIFIED_MIN_ETH 👍